plugins/autocast/package.json

```diff
@@ -31,7 +31,7 @@
     "check": "tsc ./**/*.ts --noEmit --esModuleInterop",
     "test": "jest ./**/*.spec.ts --config=../../jest.config.js --runInBand"
   },
-  "keywords": [],
+  "keywords": ["flatfile-plugins", "category-extractors", "featured"],
```

Comment:
`category-extractors` doesn't seem appropriate for autocast. Maybe `category-transform` to match the Guides?

Reply:
thanks oops